Formula/cargo-udeps.rb

```diff
@@ -25,10 +29,29 @@ class CargoUdeps < Formula
   end
 
   def install
-    system "cargo", "install", *std_cargo_args
+    ENV["LIBGIT2_SYS_USE_PKG_CONFIG"] = "1"
+    ENV["LIBSSH2_SYS_USE_PKG_CONFIG"] = "1"
+    ENV["OPENSSL_DIR"] = Formula["openssl@1.1"].opt_prefix
+    ENV["OPENSSL_NO_VENDOR"] = "1"
+    system "cargo", "install", "--no-default-features", *std_cargo_args
+  end
+
+  def check_binary_linkage(binary, library)
+    binary.dynamically_linked_libraries.any? do |dll|
+      next false unless dll.start_with?(HOMEBREW_PREFIX.to_s)
+
+      File.realpath(dll) == File.realpath(library)
+    end
   end
```

Comment:
Can this be a function in brew instead?

Reply:
I think it could be. Unfortunately I'm familiar enough with neither Ruby nor your codebase to make this decision... I'll be glad if you could help :)

Reply:
(Besides, I believe this is not in the scope of my original patch so I have little clue about what I'm trying to do here...)

Reply:
Yea, it should be a function in `brew`, but I just haven't gotten around to moving it there. It's on my to-do list.